.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Backup files generated by rustfmt
 **/*.rs.bk
+
+# IDEA directory
+.idea/
```

Comment:
Please remove this, it is not relevant to the project.

If you want to ignore specific files/folders but do not edit `.gitignore`, you can use `.git/info/exclude` file.

Reply:
my bad, sorry

Reply:
I use Clion to write rust, and it create this stupid directory in the project root to save the project settings and configuration, and some times I forget about it and I run `git add . && git commit -Sm "message"`